### PR TITLE
LIVE-2180 feat/v3 portfolio polish

### PR DESCRIPTION
**Pull request by LFBarreto**
Update to portfolio page UI according to user feedbacks + polish on graphs

https://user-images.githubusercontent.com/11752937/165937486-5f1ba277-2a90-4e30-8b0e-1af0f3588dcb.mp4

Screenshot_2022-04-29-12-11-03-685_com ledger live debug

Screenshot_2022-04-29-12-16-56-874_com ledger live debug
Screenshot_2022-04-29-12-15-53-765_com ledger live debug

### Type

UI Polish... -->

### Context

[LIVE-2180]
[LIVE-2187]
[LIVE-2195]
[LIVE-2201]
[LIVE-2206]
[LIVE-2199]
[LIVE-2195]

### Parts of the app affected / Test plan

[LIVE-2180]: https://ledgerhq.atlassian.net/browse/LIVE-2180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-2187]: https://ledgerhq.atlassian.net/browse/LIVE-2187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2195]: https://ledgerhq.atlassian.net/browse/LIVE-2195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2201]: https://ledgerhq.atlassian.net/browse/LIVE-2201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2206]: https://ledgerhq.atlassian.net/browse/LIVE-2206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2199]: https://ledgerhq.atlassian.net/browse/LIVE-2199?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-2195]: https://ledgerhq.atlassian.net/browse/LIVE-2195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by valpinkman — APPROVED** [review]
Code changes look good to me.

Great to have the Graphs back apparently 😏

**Review by thomasrogerlux — APPROVED** [review]
Looks good

**Comment by ofreyssinet-ledger** [comment]
[LIVE-2201] handled by https://github.com/LedgerHQ/ledger-live-mobile/pull/2455/commits/5603d63327ee3b7a8b1d1a3b6406b52f9ad304eb
[LIVE-2206] handled by https://github.com/LedgerHQ/ledger-live-mobile/pull/2455/commits/0110f4fd80ee9a919cfbbd6a5c81408b9ca62b81

thanks @cgrellard-ledger 👍

**Comment by ofreyssinet-ledger** [comment]
[LIVE-2199] and [LIVE-2195] to be addressed by adding an arrow & always displaying the delta/difference in countervalue on the graph

File: src/components/Delta.tsx [hunk 1]
```diff
@@ -42,35 +42,36 @@ function Delta({
...
+      : ["neutral.c100", () => null, ""];
+
   if (
     percent &&
     ((valueChange.percentage === 0 && !show0Delta) ||
       valueChange.percentage === null ||
       valueChange.percentage === undefined)
   ) {
     if (fallbackToPercentPlaceholder) return percentPlaceholder;
+    if (percent && ArrowIcon) return <ArrowIcon size={16} color={color} />;
```
Comment (ofreyssinet-ledger) [inline src/components/Delta.tsx:64]: `!!ArrowIcon` is always true no ? even if it's `() => null`

Comment (cgrellard-ledger) [reply src/components/Delta.tsx:64]: yes, I kept the logic below at line 77 in prevention for further modifications where we would not put a function but just a null value.
But currently yet it's useless I can remove it if you want

**Comment by ofreyssinet-ledger** [comment]
Android build: https://github.com/LedgerHQ/ledger-live-build/actions/runs/2263433219
iOS build: https://github.com/LedgerHQ/ledger-live-build/actions/runs/2263439179

**Review by ychen-ledger — APPROVED** [review]
Validated with: 

- LLM 3.0.8
- Pixel 4 Android 12/ iPhone 11 iOS 14.7
- ETH/BTC accounts
- All subtask closed

**Pull request merged**